### DISCUSSION TITLE
Format place holder values

### DISCUSSION THREAD

**Discussion by joeo73-impinj** [ember-intl/ember-intl#1615] [Q&A] [answered]
In my intl yaml I have the following line:

lessThanOrEqualTo: '{description} must be less than or equal to {lte}'

Is there a way to format the {lte} placeholder?  Sort of like:

lessThanOrEqualTo: '{description} must be less than or equal to {lte?formatNumber}' ?

**Comment by sukima** [comment, marked as answer]
Do you mean `{description} must be less than or equal to {lte, number}`?

https://formatjs.io/docs/core-concepts/icu-syntax/#number-type

**Reply by joeo73-impinj** [reply]
Yes, thank you.  This works?  I didn't see it documented: https://ember-intl.github.io/ember-intl/docs/helpers/format-number

**Reply by sukima** [reply]
Yes, It should. ember-intl uses format.js under the hood.

**Reply by ijlee2** [reply]
@joeo73-impinj `ember-intl` follows the ICU message syntax, which is mentioned in [Overview](https://ember-intl.github.io/ember-intl/versions/v7.0.6/docs) and [`{{t}}` helper - Passing data](https://ember-intl.github.io/ember-intl/versions/v7.0.6/docs/helpers/t#passing-data). For more information about syntax, you can visit these two places:

- https://formatjs.io/docs/core-concepts/icu-syntax/
- https://unicode-org.github.io/icu/userguide/format_parse/messages/
